boards/v2/raytac/raytac_mdbt53v_db_40/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if(("${BOARD_IDENTIFIER}" STREQUAL "/nrf5340/cpuapp" OR "${BOARD_IDENTIFIER}" STREQUAL "/nrf5340/cpuapp/ns")
+   AND CONFIG_BOARD_ENABLE_CPUNET)
```

this is board specific CMakeLists.txt, and only sourced when this board is building, so we could reduce check to:
```suggestion
if(CONFIG_SOC_NRF5340_CPUAPP_QKAA AND CONFIG_BOARD_ENABLE_CPUNET)
```

(not required for merging, but could be considered in case file has to be touched for other reasons)
Same comment goes for other cmake files in commit.